app/controllers/admin/schedules_controller.rb

```diff
@@ -7,10 +7,11 @@ class SchedulesController < Admin::BaseController
     load_resource :venue, through: :conference, singleton: true
 
     skip_before_filter :verify_authenticity_token, only: [:update]
-    layout 'schedule'
 
     def show
-      authorize! :update, @program.events.new
+      event = @program.events.new
```

Comment:
Sorry, missed to expand the code ...

However, this looks to me like it belongs to a create action:

```
unless @program.selected_schedule.present?
  schedule = @program.schedules.create
  @program.selected_schedule = schedule.id
  @program.save!
end
@schedule_id = @program.selected_schedule
```